src/LessMsi.Gui/MainForm.cs

```diff
@@ -722,13 +736,13 @@ private void InitializeComponent()
             this.exitToolStripMenuItem});
             this.fileToolStripMenuItem.Name = "fileToolStripMenuItem";
             this.fileToolStripMenuItem.Size = new System.Drawing.Size(37, 20);
-            this.fileToolStripMenuItem.Text = "&File";
+            this.fileToolStripMenuItem.Text = $"&{Strings.File}";

```

Comment:
I think the `&` should be added to the translation itself here - maybe rename it to `FileMenuLabel` or something if it is confusing to have "File" there. The I doubt the first letter will make sense in all languages as the keyboard accelerator.

Reply:
added ```FileMenuLabel``` string as you requested